src/components/layout/addons/AddonsLayout.js

```diff
@@ -0,0 +1,257 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import pluralize from 'pluralize';
+import { Input, TableOfContents, global, styles, TagList, TagLink } from '@storybook/design-system';
+import { graphql } from 'gatsby';
+import GatsbyLinkWrapper from '../../basics/GatsbyLinkWrapper';
+import { AddonsLearn } from './AddonsLearn';
+import { AddonsList } from './AddonsList';
+import { AddonsAside, AddonsAsideTitle } from './AddonsAside';
+
+const { breakpoint, spacing, color, pageMargins, typography } = styles;
+const { GlobalStyle } = global;
+
+const Content = styled.main`
+  flex: 1 1 auto;
+  min-width: 0; /* do not remove  https://weblog.west-wind.com/posts/2016/feb/15/flexbox-containers-pre-tags-and-managing-overflow */
+`;
+
+const Wrapper = styled.div`
+  ${pageMargins}
+  padding-bottom: 3rem;
+
+  @media (min-width: ${breakpoint * 1.333}px) {
+    padding-top: 4rem;
+    padding-bottom: 4rem;
+    display: ${(props) => (props.searchLayout ? 'block' : 'flex')};
+  }
+`;
+
+const Divider = styled.div`
+  border-top: 1px solid ${color.border};
+  margin-top: 24px;
+  margin-bottom: 24px;
+`;
+
+const Sidebar = styled.div`
+  flex: 1;
+  margin: 1rem 0 2rem;
+  padding-bottom: 1rem;
+  border-bottom: 1px solid ${color.border};
+  display: ${(props) => (props.hideSidebar ? 'none' : 'block')};
+  margin-bottom: 32px;
+
+  @media (min-width: ${breakpoint * 1.333}px) {
+    flex: 0 0 240px;
+    margin-top: 0;
+    margin-top: 0;
+    margin-left: auto;
+    margin-right: ${(props) => (props.searchLayout ? 'auto' : '20px')};
+    margin-bottom: ${(props) => (props.searchLayout ? spacing.padding.medium : 0)}px;
+    padding-bottom: 0;
+    padding-right: ${(props) => (props.searchLayout ? 0 : 20)}px;
+    border-bottom: none;
+  }
+`;
+
+Sidebar.propTypes = {
+  hideSidebar: PropTypes.bool.isRequired,
+};
+
+const ToCContent = styled.div`
+  /* Hide ToC on mobile, the primary navigation is search */
+  display: none;
+
+  ${(props) =>
+    props.hideToC
+      ? `
+          display: none;
+        `
+      : `
+          @media (min-width: ${breakpoint * 1.333}px) {
+            display: block;
+            margin-top: 1.5rem;
+          }
+        `}
+`;
+
+ToCContent.propTypes = {
+  hideToC: PropTypes.bool.isRequired,
+};
+
+const SearchContainer = styled.div`
+  display: flex;
+`;
+
+const StyledAddonsList = styled(AddonsList)`
+  flex: 1 1 auto;
+`;
+
+const RelatedTagsList = styled(TagList)`
+  margin-bottom: 48px;
+`;
+
+const SearchInput = styled(Input)`
+  flex: 1 1 auto;
+
+  #addons-search {
+    font-size: ${typography.size.s2}px;
+    padding-left: 40px;
+    padding-top: 12px;
+    padding-bottom: 12px;
+  }
+
+  svg {
+    left: 16px;
```

Comment:
```suggestion
    left: 16px;
    font-size: ${typography.size.s2}px;
```

This tweaks makes the icon size larger